packages/opam-create/opam-create.0.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "nv-vn <nv@cock.li>"
+authors: "nv-vn <nv@cock.li>"
+homepage: "https://github.com/nv-vn/opam-create"
+bug-reports: "https://github.com/nv-vn/opam-create/issues"
+license: "GPL"
+dev-repo: "https://github.com/nv-vn/opam-create.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: ["cp" "./main.native" "%{bin}%/opam-create"]
+remove: ["rm" "%{bin}%/opam-create"]
```

Comment:
if you add a `tags: [ "flags:plugin" ]` , it will also prompt the user to install the plugin if they type `opam create` and it isnt already present.